djpaddle/utils.py

```diff
@@ -1,43 +1,30 @@
 import base64
-import hashlib
 import collections
-import phpserialize
 
+import phpserialize
 from Crypto.PublicKey import RSA
-from django.core.exceptions import ImproperlyConfigured
 
 try:
     from Crypto.Hash import SHA1
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
I'm not a fan of `pragma: no cover` generally but I wasn't sure of another way to test this?